server/routerlicious/packages/routerlicious-base/src/riddler/runnerFactory.ts

```diff
@@ -47,6 +50,25 @@ export class RiddlerResources implements IResources {
 
 export class RiddlerResourcesFactory implements IResourcesFactory<RiddlerResources> {
     public async create(config: Provider): Promise<RiddlerResources> {
+        // Cache connection
+        const redisConfig = await config.get("redis");
```

Comment:
When we added Throttling logic, with a Redis cache, we added an additional "redisForThrottling" config. In production, we still use the same values as the other Redis configs, but it leaves the option open for adding a new Redis instance for that specific usage. I'm wondering if we should do the same here?

Reply:
Added redisForTenantCache configs

Reply:
@zhangxin511 Suggested we don't use redisForTenantCache

Reply:
We decided to keep redisForTenantCache